Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SEC102_001.EmailAddress.sarif

```diff
@@ -82,9 +82,9 @@
           "fingerprints": {
             "SecretHashSha256/current": "a4965ae12f3568daff154d92dc41cddff82ae7b6674138279bbdac60217d7d1b",
             "AssetFingerprint/current": "{\"host\":\"example.com\",\"id\":\"email\"}",
+            "ValidationFingerprintHashSha256/current": "6064325b0dd768a320862c00c981779e3bccd9a3fa3ce9dc1473a9a41f188f53",
```

Comment:
>ValidationFingerprintHashSha256 [](http://example.com/codeflow?start=13&length=31)

This is unfortunate, changes in code have resulted in reordered dictionary keys. We should consider using a sorted dictionary to preserve a deterministic order.